tekton/release-pipeline-run.yml

```diff
@@ -1,24 +1,28 @@
 ---
-apiVersion: tekton.dev/v1alpha1
-kind: PipelineResource
+apiVersion: v1
+kind: PersistentVolumeClaim
```

Comment:
Mid-term, we should use `volumeClaimTemplate` for the workspace (and thus not having to declare this PVC)